roles/common/vars/main.yml

```diff
@@ -24,3 +24,7 @@ confluent_yum_repositories:
     gpgcheck: "{{confluent_common_repository_redhat_clients_gpgcheck}}"
     gpgkey: "{{confluent_common_repository_redhat_clients_gpgkey}}"
     enabled: "{{confluent_common_repository_redhat_clients_enabled}}"
+
+### Pip packages to be installed on managed nodes
+pip_packages:
```

Comment:
the variable which we don't expect to be changed ( generally) should go in common/defaults/main

Reply:
thanks @nsharma-git for checking, 

yeah, I was confused too, all variables in <role>/defaults/main.yml go in VARIABLES.md documentation. Are we okay with that ? we don't want users to update this list of pip_packages, right ? 🤔

Reply:
Yes, this is correct. Sorry, I overlooked it.